docs/www/data-transform.md

```diff
@@ -0,0 +1,393 @@
+
+## Clean up the transform
+
+To stop a transform, you have to remove the transform from the cluster brokers.
+
+For our sample transform, run: `rpk wasm remove uppercase`
+
+The transform stops processing events and is removed.
+
+## Source code
+
+You can check what are all the possible export values in our [GitHub Page](https://github.com/vectorizedio/redpanda/blob/dev/src/js/modules/public/Coprocessor.ts).
+
+The (API package)[https://www.npmjs.com/package/@vectorizedio/wasm-api] is published under npm.
+
+## Final considerations
+
+Currently we only support Node.js but in the future, we're planning to add support for a complete WASM engine so that it can compile and run any WebAssembly binary on the fly. 
+
+### Running untrusted code
+
+We do not recommend you to run untrusted code. The reason for this is that the service will be running with the same privileges as Redpanda and with permission to read your data. 
+
+The golden rule here is to only run code where you thrust the authors or that you wrote yourself.
+
+### Under the hood
+
+We create a global control topic for each Redpanda cluster with the name `coproceessor_internal_topic`. This is done to support any Kafka client so you don´t *need* to use `rpk`. If anyone publishes to this topic, it will invoke a deployer implicitly. You just need to publish with the same headers and the correct format that we use. 
+
+### Scaling
+
+If you execute different types of transformations in different topics it's more advantageous to have multiple deployments. Remember that every time a transformation happens, you're subscribing to a topic and producing to another. Even though this process is really fast because it happens inside Redpanda, there's still a processing overhead. 
+
+### Tech Preview Caveats
+
+We don't use [Raft](https://raft.github.io/) to publish and replicate the data. So it's possible to have inconsistencies in some cases. For example, if you have a topic with a replication factor > 1 and you have stateful coprocessors that aren't idempotent producing different data onto these replication topics. 
+
+The coprocessor currently doesn't have any system limits. For example there's not a limit in processing time, memory usage or number of topics created.
+
+The coprocessor doesn't save your state. In the case of a crash, for example, if you have a counter, you'll lose that counter. The best practice here is to always avoid stateful implementations. The only state that it's kept and checkpointed is the `PolicyInjection` variable. The semantics of your materialized topics is `At-Least-Once`. Redpanda saves the offset from wherever you were reading. In the case of a crash, your deployed code will be reprocessed, based on your `PolicyInjection` policy. If it's `PolicyInjection.Latest` for example, then it will be from whatever offset was saved before the crash.
```

Comment:
Actually is Latest is chosen it won't re-process from whatever offset was saved before the crash it will be the actual current latest offset of the input topic. Saved picks up from where it left off after a crash.

Also maybe we don't want to say the `policyInjection` variable is saved but rather what it represents which is the source topic offset.